### PR TITLE
Run CI with all feature variants

### PR DESCRIPTION
**Pull request by liff**
Addresses #77

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -40,7 +45,7 @@ jobs:
         key: $clippy-cache-${{ steps.toolchain.outputs.rustc_hash }}-${{ hashFiles('**/Cargo.lock') }}
 
     - name: "Clippy ${{ matrix.rust }}"
-      run: cargo clippy --features=rt-tokio-crypto-rust --all-targets --all -- -D clippy::dbg_macro -D warnings -F unused_must_use
+      run: cargo clippy --features=${{ feature }} --all-targets --all -- -D clippy::dbg_macro -D warnings -F unused_must_use
```
Comment (complexspaces) [inline .github/workflows/ci.yaml:48]: You'll need to use `matrix.feature` here (and in the other places) instead, like what is already done for `matrix.rust` in the toolchain setup:
```suggestion
      run: cargo clippy --features=${{ matrix.feature }} --all-targets --all -- -D clippy::dbg_macro -D warnings -F unused_must_use

```

**Review by complexspaces — CHANGES_REQUESTED** [review]
Thanks for picking this one up 😄.

It doesn't look like the current revision is a valid GHA definition file, and I've called out where the change needs to be made. 

CI didn't setup to run on your push because it failed to parse:
image

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (liff) [reply .github/workflows/ci.yaml]: Fixed, sorry about that.

Comment (complexspaces) [reply .github/workflows/ci.yaml]: No problem!

**Review by complexspaces — APPROVED** [review]

**Pull request merged**